packages/suite/src/constants/suite/routes.ts

```diff
@@ -15,6 +15,7 @@ type RouteParamsTypes = {
     accountIndex: number;
     accountType: NonNullable<Network['accountType']>;
     cancelable: boolean;
+    contractAddressPrefill?: string;
```

Comment:
Was thinking about some better name for this but didn't come up with anything.

Reply:
how about just `contractAddress`?

Reply:
there is symbol, accountType, i think it is fine to use contractAddress